po/pt.po

```diff
@@ -0,0 +1,361 @@
+
+#: src/Widgets/ToolBars/TextToolBar.vala:117
+msgid "Insert link"
+msgstr "Inserir link"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:119
+msgid "Insert image"
+msgstr "Inserir imagem"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:121
+msgid "Insert table"
+msgstr "Inserir tabela"
+
+#: src/Widgets/ActionBar.vala:47
+#, c-format
+msgid "%d character with spaces"
+msgid_plural "%d characters with spaces"
+msgstr[0] "%d caractere com espaços"
+msgstr[1] "%d caracteres com espaços"
+
+#: src/Widgets/ActionBar.vala:61
+#, c-format
+msgid "%d character without spaces"
+msgid_plural "%d characters without spaces"
+msgstr[0] "%d caractere sem espaços"
+msgstr[1] "%d caracteres sem espaços"
+
+#: src/Widgets/PreferenceDialog.vala:27 src/Widgets/TitleBar.vala:62
+#: src/Widgets/TitleBar.vala:68
+msgid "Preferences"
+msgstr "Preferências"
+
+#: src/Widgets/PreferenceDialog.vala:33
+msgid "Document folder location:"
+msgstr "Localização da pasta Documentos:"
+
+#: src/Widgets/PreferenceDialog.vala:37
+msgid "Select Document Folder…"
+msgstr "Selecione a pasta do documento..."
+
+#: src/Widgets/PreferenceDialog.vala:43
+msgid "Include spaces to count characters:"
+msgstr "Incluir espaços na contagem de caracteres"
+
+#: src/Widgets/PreferenceDialog.vala:64
+msgid "Close"
+msgstr "Fechar"
+
+#. /TRANSLATORS: The string shown in the titlebar. "Writer" is the name of this app.
+#: src/Widgets/TitleBar.vala:31 src/MainWindow.vala:78
+msgid "Writer"
+msgstr "Writer"
+
+#: src/Widgets/TitleBar.vala:38
+msgid "Open a file"
+msgstr "Abrir um arquivo"
+
+#: src/Widgets/TitleBar.vala:41
+msgid "Save this file with a different name"
+msgstr "Salvar esse arquivo com outro nome"
+
+#: src/Widgets/TitleBar.vala:44
+msgid "Restore this file"
+msgstr "Restaurar arquivo"
+
+#: src/Widgets/TitleBar.vala:47
+msgid "Print this file"
+msgstr "Imprimir arquivo"
+
+#: src/Widgets/TitleBar.vala:51
+msgid "Find"
+msgstr "Encontrar"
+
+#: src/Widgets/TitleBar.vala:57
+msgid "Undo"
+msgstr "Desfazer"
+
+#: src/Widgets/TitleBar.vala:60
+msgid "Redo"
+msgstr "Refazer"
+
+#: src/Application.vala:198
+#, c-format
+msgid "Untitled Document %i"
+msgstr "Documento Sem Título %i"
+
+#: src/Application.vala:219
+msgid "Choose a file to open"
+msgstr "Escolha um arquivo para abrir"
+
+#: src/Application.vala:221 src/Application.vala:296
+msgid "Rich Text Format (.rtf)"
+msgstr "Rich Text Format (.rtf)"
+
+#: src/Application.vala:224 src/Application.vala:299
+msgid "Cancel"
+msgstr "Cancelar"
+
+#: src/Application.vala:225
+msgid "Open"
+msgstr "Abrir"
+
+#: src/Application.vala:291
+msgid "Save file with a different name"
+msgstr "Salvar arquivo com outro nome"
+
+#: src/Application.vala:300
+msgid "Save"
+msgstr "Salvar"
+
+#: src/Application.vala:326
+msgid "Are you sure you want to revret this file?"
+msgstr "Você tem certeza que quer reverter esse arquivo?"
+
+#: src/Application.vala:327
+msgid "Changes you made will be discarded."
+msgstr "As mudanças que você fez serão descartadas"
```

Comment:
Is this intentional that the trailing period that is in the source string missing?